File: 프로그래머스/lv1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.c
#include <stdio.h>
#include <stdbool.h>
#include <string.h>
#include <stdlib.h>

char* solution(const char* phone_number) {
    int length = strlen(phone_number);
    char* answer = (char*)malloc(sizeof(char)*length + 1);
    strcpy(answer, phone_number);
    for(int i=0; i<length - 4; i++){
        answer[i]='*';
    }
    return answer;
}

File: 프로그래머스/lv1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cs
public class Solution {
    public string solution(string phone_number) {
        string str = "";
        for (int i = 0; i < phone_number.Length; i++) {
            if(i < phone_number.Length - 4)
            {
                str += "*";
            }
            else
            {
                str += phone_number[i];
            }
        }
    return str;
    }
}